<ui version="4.0" >
 <class>ResourceAssignmentView</class>
 <widget class="QWidget" name="ResourceAssignmentView" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>451</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>ResourceAssignmentView</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="0" column="0" >
    <widget class="QSplitter" name="m_splitter" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QTreeWidget" name="m_resList" >
      <column>
       <property name="text" >
        <string>Name</string>
       </property>
      </column>
      <column>
       <property name="text" >
        <string>Type</string>
       </property>
      </column>
     </widget>
     <widget class="QTreeWidget" name="m_taskList" >
      <column>
       <property name="text" >
        <string>Tasks</string>
       </property>
      </column>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
